mercadopago/px-ios
# (Added)[PXN-3971] OneTap button logic when having ReviewAndConfirm

## What has changed?

When having to show ReviewAndConfirm, button in OneTap, CCV and Offline Payment Methods VCs should read **Continue** rather than **pay** (TBD actual translations in ESP/BR). 

Also added _Advanced Configuration_ in PXSecurityCodeViewModel, decoupled method for creating the button in each VC, and fixed flow for Offline Payment Methods ([PR](https://github.com/mercadopago/px-ios/pull/2777)).

- **TODOs** in code will be addressed in later tickets. 

## How to test:

- Add a builder integration with an Advanced Configuration, setting _CustomReviewAndConfirmViewController_ as the injected VC. 
- Check for buttons in OneTapVC, SecurityCodeVC, OfflineMethodsVC to read **Continuar**.

## Prints for layout changes:

![buttonSelection](https://user-images.githubusercontent.com/61248081/171302185-bca88eed-d7ca-432a-9613-3cc607253a9f.gif)
